homeassistant/components/tessie/sensor.py

```diff
@@ -24,20 +25,31 @@
     UnitOfTemperature,
     UnitOfTime,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import StateType
+from homeassistant.util import dt as dt_util
 
 from .const import DOMAIN
 from .coordinator import TessieStateUpdateCoordinator
 from .entity import TessieEntity
 
 
+@callback
+def hours_to_datetime(value: StateType) -> datetime | None:
```

Comment:
Totally forgotten, please prefix functions with `async_` if they are safe to run in the event loop.